### DISCUSSION TITLE
Version v1.0.0 Release

### DISCUSSION THREAD

**Discussion by hanisirfan** [seajell/seajell#25] [Announcements]
I'm happy to announce that the first release of SeaJell is now out.
Features included:
- User system / authentication
- Add and remove certificate and event
- Generate signature image
For now the system is only in Malay. Might add English support in the future.

<hr /><em>This discussion was created from the release <a href='https://github.com/hanisirfan/seajell/releases/tag/v1.0.0'>Version v1.0.0 Release</a>.</em>

**Comment by airi-miraiu** [comment]
nicer
